mu-plugins/blocks/global-header-footer/admin-bar.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace WordPressdotorg\MU_Plugins\Global_Header_Footer;
+
+use Rosetta_Sites, WP_Post, WP_REST_Server, WP_Theme_JSON_Resolver;
+
+defined( 'WPINC' ) || die();
+
+/* Actions & filters */
+add_action( 'admin_bar_menu', __NAMESPACE__ . '\filter_admin_bar_links', 500 ); // 500 to run after all items are added to the menu.
+add_filter( 'show_admin_bar', __NAMESPACE__ . '\should_show_admin_bar' );
+
+/**
+ * Always show the admin bar.
+ *
+ * @param bool $show_admin_bar Whether the admin bar should be shown.
+ * @return bool
+ */
+function should_show_admin_bar( $show_admin_bar ) {
+	// @todo Don't enable on the login page.
+	return true;
+}
+
+/**
+ * Filter the admin bar links to simplify the frontend view.
+ *
+ * @param \WP_Admin_Bar $wp_admin_bar The WP_Admin_Bar instance, passed by reference.
+ */
+function filter_admin_bar_links( $wp_admin_bar ) {
+	// Only change the admin bar on the frontend.
+	if ( is_admin() ) {
+		return;
+	}
+
+	if ( is_user_logged_in() ) {
+		// Remove all items in these submenus.
+		$parent_remove_list = [ 'wp-logo-external', 'wp-logo', 'site-name', 'appearance' ];
+
+		// Remove all these items (includes the top-level items from the parent list).
+		$remove_list = array_merge(
+			$parent_remove_list,
+			[ 'comments', 'stats', 'search', 'my-sites', 'admin-bar-likes-widget' ]
+		);
```

Comment:
Specifically removing elements like this ensures that items added by plugins (debug, query monitor, copy post, etc) are not removed.